### DISCUSSION TITLE
Does rocm support rx6500m on windows?

### DISCUSSION THREAD

**Discussion by adityaprakashgupta** [ROCm/ROCm#2364] [Q&A] [answered]
My laptop is having rx6500m installed does rocm support navi 24 and gfx1034?

**Comment by taylding-amd** [comment, marked as answer]
Hi @adityaprakashgupta, unfortunately, gfx1034 is not supported, you can refer to the [Windows-supported GPUs](https://rocm.docs.amd.com/projects/install-on-windows/en/latest/reference/system-requirements.html) section in our official document for more information.
